### PR TITLE
[Enhance] Provide offline annotations and polish docs

### PR DESCRIPTION
**Pull request by JingweiZhang12**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/zh_cn/user_guides/dataset_prepare.md [hunk 1]
```diff
@@ -75,6 +75,18 @@ mmdetection3d
 
 ## 数据下载和预处理
 
+### 数据集标注文件列表
+
```
Comment (sunjiahao1999) [inline docs/zh_cn/user_guides/dataset_prepare.md:79]: Add a description here to illustrate the purpose of prepared `info.pkl` and I suggest putting the list at the end of `dataset_prepare.md`.

File: README.md [hunk 2]
```diff
@@ -66,7 +66,7 @@ Documentation: https://mmdetection3d.readthedocs.io/
 
 English | [简体中文](README_zh-CN.md)
 
-The master branch works with **PyTorch 1.6+**.
+The main branch works with **PyTorch 1.8+**.
```
Comment (Xiangxu-0103) [inline README.md:69]: Why not support PyTorch 1.6? Is there any reference?

File: docs/en/user_guides/dataset_prepare.md [hunk 3]
```diff
@@ -102,19 +102,39 @@ In an environment using slurm, users may run the following command instead:
...
 Note that:
 
+- If the preprocess of Waymo dataset is very slow or blocked, you could modify the `--workers` to a smaller number. If it still can't be solved, you could set the `--workers` as 0 to avoid using multiprocess.
+
 - If your local disk does not have enough space for saving converted data, you can change the `--out-dir` to anywhere else. Just remember to create folders and prepare data there in advance and link them back to `data/waymo/kitti_format` after the data conversion.
 
-- If you want faster evaluation on Waymo, you can download the preprocessed [metainfo](https://download.openmmlab.com/mmdetection3d/data/waymo/idx2metainfo.pkl) containing `contextname` and `timestamp` to the directory `data/waymo/waymo_format/`. Then, the dataset config is modified like the following:
+**Tips**:
+
+- **Ready-made Annotations**. We have provided the annotation files generated offline [here](#summary-of-annotation-files). However, the original Waymo data still needs to be converted kitti-format by yourself.
```
Comment (Xiangxu-0103) [inline docs/en/user_guides/dataset_prepare.md:129]:
```suggestion
- **Ready-made Annotations**. We have provided the annotation files generated offline [here](#summary-of-annotation-files). However, the original Waymo data still needs to be converted to kitti-format by yourself.
```

File: docs/en/user_guides/dataset_prepare.md [hunk 4]
```diff
@@ -157,6 +185,8 @@ To prepare ScanNet data, please see its [README](https://github.com/open-mmlab/m
 
 To prepare SUN RGB-D data, please see its [README](https://github.com/open-mmlab/mmdetection3d/blob/dev-1.x/data/sunrgbd/README.md).
 
+**Tips**: We have also provided data annotations generated offline [here](#summary-of-annotation-files). You could download them and place them under `data/${DATASET}/`. However, if you want to use `ObjectSample` Augmentation in LiDAR-based detection methods, you should additionally generate the groundtruth database file and annotations by adding `--only-gt-database` in the command of running `create_data.py` like [Kitti](#kitti).
```
Comment (Xiangxu-0103) [inline docs/en/user_guides/dataset_prepare.md:188]: For `S3DIS`, `ScanNet` and `SUN RGB-D` dataset, we don't need to generate groundtruth database file, but need to generate point cloud files and semantic/instance masks (if it has) here.

File: README.md (hunk 2 above)
Comment (JingweiZhang12) [reply README.md:69]: The update of MMCV: OpenMMLab 2.0 series only supports pytorch1.8+

File: docs/en/user_guides/dataset_prepare.md [hunk 5]
```diff
@@ -102,19 +102,39 @@ In an environment using slurm, users may run the following command instead:
...
+Download Waymo open dataset V1.4.1 [HERE](https://waymo.com/open/download/) and its data split [HERE](https://drive.google.com/drive/folders/18BVuF_RYJF0NjZpt8SnfzANiakoRMf0o?usp=sharing). Then put `.tfrecord` files into corresponding folders in `data/waymo/waymo_format/` and put the data split `.txt` files into `data/waymo/kitti_format/ImageSets`. Download ground truth `.bin` file for validation set [HERE](https://console.cloud.google.com/storage/browser/waymo_open_dataset_v_1_4_1/validation/ground_truth_objects) and put it into `data/waymo/waymo_format/`. A tip is that you can use `gsutil` to download the large-scale dataset with commands. You can take this [tool](https://github.com/RalphMao/Waymo-Dataset-Tool) as an example for more details. Subsequently, prepare waymo data by running:
 
 ```bash
 python tools/create_data.py waymo --root-path ./data/waymo/ --out-dir ./data/waymo/ --workers 128 --extra-tag waymo
 ```
 
 Note that:
 
+- If the preprocess of Waymo dataset is very slow or blocked, you could modify the `--workers` to a smaller number. If it still can't be solved, you could set the `--workers` as 0 to avoid using multiprocess.
```
Comment (ZCMax) [inline docs/en/user_guides/dataset_prepare.md:123]: In case the preprocessing of Waymo dataset is slow or obstructed, consider reducing the number of `--workers`. If this doesn't resolve the issue, you may set `--workers` as 0 to avoid multiprocess usage.

**Review by Xiangxu-0103 — APPROVED** [review]

File: docs/en/user_guides/dataset_prepare.md [hunk 6]
```diff
@@ -102,19 +102,39 @@ In an environment using slurm, users may run the following command instead:
...
 
-- If you want faster evaluation on Waymo, you can download the preprocessed [metainfo](https://download.openmmlab.com/mmdetection3d/data/waymo/idx2metainfo.pkl) containing `contextname` and `timestamp` to the directory `data/waymo/waymo_format/`. Then, the dataset config is modified like the following:
+**Tips**:
+
+- **Ready-made Annotations**. We have provided the annotation files generated offline [here](#summary-of-annotation-files). However, the original Waymo data still needs to be converted to kitti-format data by yourself.
+
+- **Waymo-mini**. If you just want to use a part of Waymo Dataset to verify some methods or debug quickly, you could use [Waymo-mini](https://download.openmmlab.com/mmdetection3d/data/waymo/waymo_mini_kitti_format.tar.gz) we provided. It only contains two segments in train split and one segment in val split from the original dataset. We have processed all the images, point clouds and annotations offline. What you should do is only downloading, and then unzipping the file to `data/waymo/`:
```
Comment (ZCMax) [inline docs/en/user_guides/dataset_prepare.md:131]: you could use our provided [Waymo-mini](https://download.openmmlab.com/mmdetection3d/data/waymo/waymo_mini_kitti_format.tar.gz).

Comment (ZCMax) [reply docs/en/user_guides/dataset_prepare.md:131]: All the images, point clouds and annotations have been processed offline so that you can directly download and unzip it to `data/waymo/`:

File: docs/en/user_guides/dataset_prepare.md [hunk 7]
```diff
@@ -102,19 +102,39 @@ In an environment using slurm, users may run the following command instead:
...
+
+  ```bash
+  tar -xzvf waymo_mini_kitti_format.tar.gz -C ./data/waymo
+  ```
+
+- **Faster evaluation**. If you want faster evaluation on Waymo, you can download the preprocessed [metainfo](https://download.openmmlab.com/mmdetection3d/data/waymo/idx2metainfo.pkl) containing `contextname` and `timestamp` to the directory `data/waymo/waymo_format/`. Then, the dataset config is modified like the following:
```
Comment (ZCMax) [inline docs/en/user_guides/dataset_prepare.md:137]: To accelerate the evaluation process for Waymo dataset, simply download the preprocessed [metainfo](https://download.openmmlab.com/mmdetection3d/data/waymo/idx2metainfo.pkl) which includes `contextname` and `timestamp`, and put it in the directory `data/waymo/waymo_format/`. Then modify the dataset config as follows:
